src/dev/impl/DevToys/UI/Controls/FileSelector.xaml

```diff
@@ -77,10 +77,16 @@
                 </StackPanel>
                 <TextBlock
                     Text="/"
+                    x:Name="PasteHyperlinkSeparator"
+                    x:Load="{x:Bind AllowPasteImage, Mode=OneWay}"
```

Comment:
Note the use of [Load](https://learn.microsoft.com/en-us/windows/uwp/xaml-platform/x-load-attribute) here.  It was not possible to use `BooleanToVisibilityConverter` because setting these items to `Collapsed` for `Visibility` hid them, but they still occupied space and would push the open directory button to the left.  `Load` will prevent them from being rendered in the tree so that the open directory button is center aligned properly.